configdb/hardware/migrations/0034_copy_instrument_state.py

```diff
@@ -0,0 +1,38 @@
+from django.db import migrations
```

Comment:
Decided to do this just to keep a complete historical record of the evolution of the DB.